treesitter/default.nix

```diff
@@ -0,0 +1,79 @@
+flake-utils: {
+  yarn2nix,
+  yarn2nix-moretea,
```

Comment:
I don't see the following packages used, are they leftover from testing or do mkYarnPackage use them somehow?

yarn2nix
yarn2nix-moretea
nodePackages

The following also aren't used. Tabry is ruby but only compiling shouldn't need it, unless you're including it for another reason?

ruby
bundlerEnv

I also don't see "fetchzip" used?

Reply:
No, you're right, this is all stuff leftover from when I was debugging... I'll clean it up!